src/olympia/conf/prod/settings.py

```diff
@@ -16,7 +16,7 @@
 EMAIL_BACKEND = EMAIL_URL['EMAIL_BACKEND']
 EMAIL_HOST_USER = EMAIL_URL['EMAIL_HOST_USER']
 EMAIL_HOST_PASSWORD = EMAIL_URL['EMAIL_HOST_PASSWORD']
-EMAIL_BLACKLIST = env.list('EMAIL_BLACKLIST')
+EMAIL_DENY_LIST = env.list('EMAIL_DENY_LIST')
```

Comment:
This needs to be changed by ops first before we can merge this. We already ran into problems of `-dev` got broken. cc @bqbn on this.